engine/version.go

```diff
@@ -8,36 +8,57 @@ import (
 )
 
 var (
-	EngineImageRepo = "registry.dagger.io/engine"
+	// Version holds the complete version number.
+	//
+	// Note: this is filled at link-time.
+	//
+	// - For official tagged releases, this is simple semver like x.y.z
+	// - For builds off our repo's main branch, this is the git commit sha
+	// - For local dev builds, this is a pseudo-random dagger ID of the source directory
+	Version string
 
-	Package = "github.com/dagger/dagger"
+	// MinimumEngineVersion is used by the client to determine the minimum
+	// allowed engine version that can be used by that client.
+	MinimumEngineVersion = "v0.11.7"
 
-	GPUSupportEnvName = "_EXPERIMENTAL_DAGGER_GPU_SUPPORT"
+	// MinimumClientVersion is used by the engine to determine the minimum
+	// allowed client version that can connect to that engine.
+	MinimumClientVersion = "v0.11.7"
```

Comment:
@vito this is the version check I think we want?

From the next release (v0.11.7):
- The CLI needs engine v0.11.7
- The engine needs CLI v0.11.7

Reply:
Yep sounds good!

Reply:
I guess we should add a step to `RELEASING.md` for updating this when needed

Reply:
@sipsma added to `RELEASING.md`, good shout!